.repo-metadata.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "datalabeling",
   "name_pretty": "Google Cloud Data Labeling",
-  "product_documentation": "https://cloud.google.com/data-labeling/docs/",
-  "client_documentation": "https://googleapis.dev/java/java-datalabeling/latest/index.html",
+  "product_documentation": "https://cloud.google.com/ai-platform/data-labeling/docs/",
+  "client_documentation": "https://googleapis.dev/java/google-cloud-datalabeling/latest/index.html",
+  "api_description": "",
```

Comment:
LGTM I'd add an issue to track to add this in

Reply:
Opened https://github.com/googleapis/java-datalabeling/issues/65 to track